lib/util/git/index.ts

```diff
@@ -42,6 +43,50 @@ export { setPrivateKey } from './private-key';
 // TODO: fix upstream types https://github.com/steveukx/git-js/issues/704
 const ResetMode = (simpleGit.default as any).ResetMode as typeof _ResetMode;
 
+// Retry parameters
+const retryCount = 5;
+const delaySeconds = 3;
+const delayFactor = 2;
+
+// A generic wrapper for simpleGit.* calls to make them more fault-tolerant
+export async function gitRetry<T>(gitFunc: () => Promise<T>): Promise<T> {
+  let round = 0;
+  while (round <= retryCount) {
+    if (round > 0) {
+      logger.debug({ round }, 'gitRetry round');
```

Comment:
```suggestion
      logger.debug(`gitRetry round ${round}`);
```

Reply:
@rarkins I think for log parsing it's better to log variables as metadata instead inline?

Reply:
I don't think anyone will be programmatically parsing this line, and it makes the human-readable output messier if we have metadata